### PR TITLE
Txi optics veto

### PR DESCRIPTION
**Pull request by ghalym**
## Description
FIX FFO dev name and description

Double the MR1K3 IN position's encoder count delta from 5,000 microns to 10,000 microns as per https://jira.slac.stanford.edu/browse/ECS-5650

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive comments
- [x] Test suite passes locally
- [x] Libraries are set to fixed versions and not ``Always Newest``
- [x] Code committed with ``pre-commit`` (alternatively ``pre-commit run --all-files``)

### CODE REVIEW

File: Arbiter/KFEArbiter/POUs/Fast_Faults_Evaluate.TcPOU [hunk 1]
```diff
@@ -3,37 +3,37 @@
...
         i_TypeCode := 16#3013);
 {attribute 'TcLinkTo' := 'TIIB[plc-txi-optics]^IO Inputs^FFO'}
 i_xFastFaultOut_txi_optics AT %I*:        BOOL;
 
 (*TXI OPTICS Protection Faults*)
 
-FFO_RIX_TXI_OPTICS :    FB_FastFault :=(
-        i_Desc := 'Fault when MR1K1 is IN while a TXI Optics is in (either MR1K3 and MR2K3)',
+FFO_RIX_TXI_OPTICS :    FB_FastFault :=(i_DevName := 'PMPS Arbiter',
+        i_Desc := 'Fault when MR1K1 is IN while a TXI Optics is IN (Check MR1K3 and MR2K3)',
```
Comment (ZLLentz) [inline Arbiter/KFEArbiter/POUs/Fast_Faults_Evaluate.TcPOU:33]: I like the updated names and descriptions :+1:

File: Arbiter/KFEArbiter/POUs/CurrentBPUpdate.TcPOU [hunk 2]
```diff
@@ -114,8 +114,8 @@ VAR
     ////////////////////////
           {attribute 'TcLinkTo' := 'TIIB[plc-txi-optics]^IO Inputs^MR1K3_Y_ENC'}
         i_MR1K3_ENC AT %I* : ULINT; // MR1K3 encoder from the txi k optics optics system
-        cMR1K3_IN_UpperLimit : UDINT := 108500000 + 5000000;
-        cMR1K3_IN_LowerLimit : UDINT := 108500000 - 5000000 ;
+        cMR1K3_IN_UpperLimit : UDINT := 108500000 + 10000000;
+        cMR1K3_IN_LowerLimit : UDINT := 108500000 - 10000000 ;
```
Comment (ZLLentz) [inline Arbiter/KFEArbiter/POUs/CurrentBPUpdate.TcPOU:118]: is there a reference for these encoder count changes?

Comment (nrwslac) [reply Arbiter/KFEArbiter/POUs/CurrentBPUpdate.TcPOU:118]: https://jira.slac.stanford.edu/browse/ECS-5650 I think its in the comments here.

**Review by nrwslac — APPROVED** [review]
lgtm

File: Arbiter/KFEArbiter/POUs/CurrentBPUpdate.TcPOU (hunk 2 above)
Comment (nrwslac) [reply Arbiter/KFEArbiter/POUs/CurrentBPUpdate.TcPOU:118]: Could use better documentation here, but 10,000 microns specified as delta converted nano here.

Comment (ZLLentz) [reply Arbiter/KFEArbiter/POUs/CurrentBPUpdate.TcPOU:118]: I'm going to add "Double the MR1K3 IN position's encoder count delta from 5,000 microns to 10,000 microns as per https://jira.slac.stanford.edu/browse/ECS-5650" to the PR description, then I'm going to approve.

**Review by ZLLentz — APPROVED** [review]

**Pull request merged**